agency/README.md

```diff
@@ -8,30 +8,35 @@ This specification contains a collection of RESTful APIs used to specify the dig
 
 ## Table of Contents
 
-* [register_vehicle](#register_vehicle)
-* [deregister_vehicle](#deregister_vehicle)
-* [update_vehicle_status](#update_vehicle_status)
-* [start_trip](#start_trip)
-* [end_trip](#start_trip)
-* [update_trip_telemetry](#update_trip_telemetry)
-* [service_areas](#service_areas)
+* [Register vehicle](#register_vehicle)
+* [Deregister vehicle](#deregister_vehicle)
+* [Update vehicle status](#update_vehicle_status)
+* [Start trip](#start_trip)
+* [End trip](#start_trip)
+* [Update trip telemetry](#update_trip_telemetry)
+* [Service areas](#service_areas)
 * [Event types](#Event-Types)
 * [Enum definitions](#enum-definitions)
 
-## register_vehicle
+## Register vehicle
 
 The Vehicle Registration API is required in order to register a vehicle for use in the system. The API will require a valid `provider_id` and `api_key`.
 
-Endpoint: `/register_vehicle`\
-Method: `POST`\
+Endpoint: `/vehicle/{provider_id}/{unique_id}`\
+Method: `PUT`\
```

Comment:
Why use PUT here ?
In my understanding this is the moment when the vehicle object is created on the agency side => POST on `/vehicle/{provider_id}` and the unique_id is attributed via the HTTP Response.

Reply:
My bad `UUID v4 provided by Operator to uniquely identify a vehicle` the unique_id is provided by the operator.

Reply:
But still why PUT ? I still consider it as object creation on the agency side.

Reply:
In my understanding, PUT is used for creation and/or update, POST for adding content to an existing item (ie. a board, forum, etc.)

Reply:
:+1: Right ! Thanks for the pointer !